src/styles/blocks/section3.scss

```diff
@@ -0,0 +1,31 @@
+.section_3 {
+  display: flex;
+  height: 40px;
+  width: 166px;
+  justify-content: stretch;
+}
+
+.section_3__link {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 12px 68px;
```

Comment:
The padding values seem to be arbitrary and may not scale well with different text lengths or font sizes. Instead of using fixed padding, consider using flexbox properties to center content and to allow the container to adapt to its content.